package.json

```diff
@@ -17,7 +17,7 @@
 	},
 	"sideEffects": false,
 	"engines": {
-		"node": ">=18"
+		"node": "^18.9.0 || >=20.5.0"
```

Comment:
```suggestion
		"node": "^18.19.0 || >=20.5.0"
```

Reply:
Oops sorry, I copy-pasted this from the issue. 🙈

Reply:
Fixed